Mention the initial pre-paid credit in the Quickstart guide (#95)
emnify/product-docs: docs/quickstart/getting-started.md

![Claim your free evaluation package with 3 triple-cut SIMs to kickstart your well-managed IoT journey. Includes: Free 3 triple-cut SIMs, form factor (Nano, Micro, Mini), our full coverage, and global fast shipping.](assets/portal-order-evaluation-pack.png)

You can order your free Evaluation SIM package on the emnify Portal.

> The package also includes an initial 10 EUR or USD of pre-paid credit to help you test emnify's services.

Reviewer: Actually, I question the need for including "prepaid" at all.
Grammarly is fine with this:

> This package includes a 10 EUR/USD credit for testing emnify's services.

```suggestion
This package includes a prepaid credit of 10 EUR/USD for testing emnify's services.
```

:::note
You have 60 days to use this pre-paid credit.
:::

Currently, the 3 triple-cut SIM cards can be delivered to most destinations worldwide with express shipment.

> To get your free Evaluation SIM package, [log into your emnify account](https://portal.emnify.com/sign) and follow these steps:

Reviewer: ```suggestion
To get your free Evaluation SIM package, [log in to your emnify account](https://portal.emnify.com/sign) and follow these steps:
```
**login** (noun) vs **log in** (verb) thus  [... log in to ...](https://www.express.co.uk/life-style/science-technology/996184/Gmail-sign-in-how-to-log-in-Gmail-account-easy-create-Gmail-email).

Reviewer: This is still not resolved.

Correct: *log* **in to** *your emnify account*

Perhaps these [Quora answers](https://www.quora.com/Which-one-is-correct-Log-in-to-your-account-Login-into-your-account-Log-in-your-account-or-Log-into-your-account-Ive-seen-people-using-all-of-them-and-Im-a-bit-confused) can explain it more clearly.
